packages/core-mobile/app/screens/bridge/Bridge.tsx

```diff
@@ -107,7 +108,7 @@ const Bridge: FC = () => {
 
   const { currencyFormatter } = useApplicationContext().appHook
   const isAmountTooLow = amount !== 0n && minimum && amount < minimum
-  const isAmountTooLarge = amount !== 0n && maximum && amount > maximum
+  const isAmountTooLarge = false //amount !== 0n && maximum && amount > maximum
```

Comment:
should we just remove this then if it is always false?

Reply:
oops, it was temporary for debugging. reverted.